app/controllers/concerns/hyrax/admin/users_controller_behavior.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# OVERRIDE FILE from Hyrax v2.9.0
```

Comment:
These should all follow this format:

#OVERRIDE `GEM_NAME` `GEM_VERSION` Reason
the words "FILE from" are just different enough to mess up searches

eg:
`# OVERRIDE Hyrax v2.9.0 authorize User access and set up roles drop down`